sycl/CMakeLists.txt

```diff
@@ -137,6 +138,10 @@ COMMENT "Copying SYCL headers ...")
 
 # Configure SYCL headers
 install(DIRECTORY "${sycl_inc_dir}/." DESTINATION "${LLVM_INST_INC_DIRECTORY}" COMPONENT sycl-headers)
+install(DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/xmethods"
```

Comment:
We are using deploy-* targets to install sycl tool chain. Please make sure that the file is installed as part of deploy-sycl-toolchain target (see line 273)

Reply:
Checked:

```
$ tree ./install/lib/clang/10.0.0/python
./install/lib/clang/10.0.0/python
\u2514\u2500\u2500 xmethods
    \u2514\u2500\u2500 accessors.py
```